packages/analytics/test/summing.spec.js

```diff
@@ -0,0 +1,186 @@
+import ezs from '../../core/src';
+import summing from '../src/summing';
+import runEzs from '../test-utils/runEzs';
+
+ezs.addPath(__dirname);
+
+describe('summing', () => {
+
+    beforeAll(() => {
+        ezs.use({ summing });
+    });
+
+    describe('wrong data', () => {
+        const undefinedData = [
+            undefined,
+            undefined,
+            undefined,
+            undefined
+        ];
+
+        const nullData = [
+            null,
+            null,
+            null,
+            null
+        ];
+
+        const undefinedNullData = [
+            undefined,
+            null,
+            undefined,
+            null
+        ];
+
+        const wrongKeyData = [
+            {
+                'hello': 'world',
+            },
+            {
+                'hello': 'world',
+            }
+        ];
+
+        it('should return no result when input contains undefined', async () => {
+            const result = await runEzs(ezs, undefinedData, 'summing');
+
+            expect(result).toHaveLength(0);
+        });
+
+        it.skip('should return no result when input contains null', async () => {
+            const result = await runEzs(ezs, nullData, 'summing');
+
+            expect(result).toHaveLength(0);
+        });
+
+        it.skip('should return no result when input contains undefined and null', async () => {
+            const result = await runEzs(ezs, undefinedNullData, 'summing');
+
+            expect(result).toHaveLength(0);
+        });
+
+        it('should return no result when input contains wrong key', async () => {
+            const result = await runEzs(ezs, wrongKeyData, 'summing');
+
+            expect(result).toHaveLength(0);
+        });
+    });
+
+    describe('summing, simple object', () => {
+        const simpleData =  [
+            {
+                'id': 1,
+                'value': [1, 1, 1]
+            },
+            {
+                'id': 2,
+                'value': [2, 2, 2]
+            }
+        ];
+
+        it('should sum value (id = id, value = value)', async () => {
+            const result = await runEzs(ezs, simpleData, 'summing');
+
+            expect(result).not.toBeNull();
+
+            expect(result).toHaveLength(2);
+
+            expect(result[0]).not.toBeNull();
+            expect(result[0].id).not.toBeNull();
+            expect(result[0].id).toStrictEqual(1);
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** Remove not.toBeNull expect

I assume that when the `.toStrictEqual` part is true, the result is not null.
The same for the next `.toStrictEqual`s.
‏
```suggestion
            expect(result[0].id).toStrictEqual(1);
```